app/users/user_router.py

```diff
@@ -0,0 +1,39 @@
+from fastapi import APIRouter, HTTPException, Depends
+from . import crud, models, schemas
+from sqlalchemy.orm import Session
+from ..database import SessionLocal, engine
+
+models.Base.metadata.create_all(bind=engine)
+
+
+router = APIRouter()
+
+
+def get_db():
```

Comment:
better to rename it to `get_session` and use `session` variable instead of db. 
also this generator should be moved to app/database.py